tests/test_other.py

```diff
@@ -8389,3 +8389,20 @@ def test_js_optimizer_parse_error(self):
                                         ^
 ''', output.stderr)
 
+  def test_wasm_sourcemap(self):
+    shutil.copyfile(path_from_root('tests', 'other', 'wasm_sourcemap', 'no_main.c'), 'no_main.c')
+    wasm_map_cmd = [PYTHON, path_from_root('tools', 'wasm-sourcemap.py'),
+                    '-s', '--prefix', '=wasm-src:///',
+                    '--llvm-dwarfdump', path_from_root('tests', 'other', 'wasm_sourcemap', 'foo.wasm.dump'),
+                    path_from_root('tests', 'other', 'wasm_sourcemap', 'foo.wasm'),
+                    'a.out.wasm.map']
+    subprocess.check_call(wasm_map_cmd)
+    open('no_main.c').read()
```

Comment:
This line doesn't seem to do anything, do we even need to copy the file?

Reply:
Removed. The copy is needed since if want to test original source inclusion into the result wasm source map.